action.yml

```
@@ -55,7 +55,10 @@ runs:
     - name: Run maven
       run: |-
         test -z "${{ inputs.run-tests }}" && SKIP_TESTS="-DskipTests"
-        mvn -B package ${SKIP_TESTS}
+      mvn -B ${{ maven-additional-options:
+    description: Any extra Maven options
+    required: false
+    default: "" }} ...
```

I think #9 specifially calls for extra options for the "non-release" invocation of maven.

However, this input can be defined just in the same way as `maven-release-options`.

```suggestion
      mvn -B ${{ inputs.maven-additional-options }} package ${SKIP_TESTS}
```